### PR TITLE
[MercatorTask & DataNormalizer] Go Glide support

### PR DESCRIPTION
**Pull request by jpopelka**
Related to https://github.com/openshiftio/openshift.io/issues/1618

### CODE REVIEW

**Comment by jpopelka** [comment]
Tested with go:github.com/Masterminds/glide:3e13fd16ed5b0808ba0fb2e4bd98eb325ccde0a1

File: f8a_worker/data_normalizer.py [hunk 1]
```diff
@@ -442,6 +442,52 @@ def _handle_gofedlib(self, data):
...
+            # [{"package": "github.com/Masterminds/vcs", "version": "^1.2.0"},
+            #  {"package": "github.com/codegangsta/cli", "version": "f89effe8"}]
+            # to
+            # ["github.com/Masterminds/vcs ^1.2.0",
+            #  "github.com/codegangsta/cli f89effe81c1ece9c5b0fda359ebd9cf65f169a51"]
+            dependencies = []
+            for dep in import_list:
+                _dep = "{name} {version}".format(name=dep['package'],
+                                                 version=dep.get('version', '')).strip()
+                # TODO: there might be also dep['subpackages'], do we want to include them too ?
```
Comment (abs51295) [inline f8a_worker/data_normalizer.py]: I think we want dep['subpackages'] as well.

File: tests/data/dataNormalizer/go-glide-from-mercator [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      {
+         "time": "2018-03-09T14:19:57.279045087+01:00",
+         "path": "/github.com/Masterminds/glide/glide.yaml",
+         "ecosystem": "Go-Glide",
+         "result": {
+            "_dependency_tree_lock_file": {
+               "hash": "7212c93931b28dba262f6e96671fc435d2562b8f91fad6ec10eacdf54607b4aa",
+               "import": [
+                  {
+                     "name": "github.com/andybalholm/cascadia",
```
Comment (abs51295) [inline tests/data/dataNormalizer/go-glide-from-mercator]: Where is this package coming from? I don't see it in glide.lock file.

File: tests/data/dataNormalizer/go-glide-from-mercator [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+                     "version": "349dd0209470eabd9514242c688c403c0926d266"
+                  },
+                  {
+                     "name": "github.com/tdewolff/parse",
```
Comment (abs51295) [inline tests/data/dataNormalizer/go-glide-from-mercator]: Same goes for this as well and others which aren't in glide.lock file.

File: f8a_worker/data_normalizer.py (hunk 1 above)
Comment (jpopelka) [reply f8a_worker/data_normalizer.py]: See latest commit.

**Pull request merged**